create_jenkins_job.py

```diff
@@ -77,7 +77,7 @@ def main(argv=None):
         'use_osrf_connext_debs_default': 'false',
         'use_fastrtps_default': 'true',
         'use_opensplice_default': 'false',
-        'ament_build_args_default': '--parallel --cmake-args -DSECURITY=ON --',
+        'ament_build_args_default': '--parallel --cmake-args -DSECURITY=ON',
```

Comment:
I think this should stay, otherwise what happens to the next argument added ? it will be considered part of the cmake args ?